Crucible.NPCs/CrucibleNpcAppearanceConfig.cs

```diff
@@ -64,23 +64,9 @@ public class CrucibleNpcAppearanceConfig
         /// <param name="npc">The NPC to apply the configuration to.</param>
         public void ApplyAppearance(CrucibleNpcTemplate npc)
         {
-            npc.Appearance.Clear();
```

Comment:
Removed this clear to allow for people to copy most of an NPC while only specifying appearance elements they want to change.